### DISCUSSION TITLE
LIB, Bootstrap et notation

### DISCUSSION THREAD

**Discussion by Jireck-npds** [npds/npds_dune#539] [Ideas]
Hello 
Toujours suite a mes testes et deboires, je vous re re  re soumet l'idée de mieux architecturé le repertoire lib

Actuel 
<img width="204" alt="image" src="https://github.com/npds/npds_dune/assets/16939826/9b5e4dc1-db06-4a66-bf17-0e9141ed5d30">

Il serait bien de ranger ce qui va ensemble et de N° ce qui doit etre numeroté.
Bootstrap avec ses plugin dans bootstrap avec son nom sans bootstrap
et la même chose avec Jquery 
et pareil avec les fonts

Par exemple : 
Lib 

- bootstrap
- >N° de version ( 5.2 / ou 5.3)
-  - > css
- - > js
- > table
- - > N° de version
- - - > css
- - - > js
- > datepicker
- - > N° de version
- - - > css
- - - > js
- css
- js
- JQUERY
- >N° de version ( 5.2 / ou 5.3)
-  - > css
- - > js
- > PLUGIN A
- - > N° de version
- - - > css
- - - > js
- >  PLUGIN B
- - > N° de version
- - - > css
- - - > js

Voila c'est tout dis ... 
cela permettait de debugger plus rapidement les users,
de mettre a jour plus simplement le code  et pour les tests du genre on colle dans bootstrap la version d'apres avec son N° il y a 2 version et dans le code on peux basculer facilmenet de l'un a l'autre en un fois que c'est bon supprimer une version.
Voir aussi les PB entre bootstrap / jquery et ses plugin etC...

**Comment by jipexu** [comment]
Certes on peut encore peaufiner la hiérarchie des répertoires mais je ne suis pas pour les numéro de version (car cela fait changer tous les appels à chaque mise à jour (qui sont très nombreuses vu le nombre de librairie et sous librairies) ce qui deviens un véritable casse tête et perte de temps ....

**Reply by Jireck-npds** [reply]
Ok meme si tu met pas le N° de version une architecture comme ca te permet de savoir qui depends de quoi et avoir une classification.
c'est mon point de vue
